### DISCUSSION TITLE
Add IMIS support

### DISCUSSION THREAD

**Discussion by thenexus00** [umbraco/Umbraco.Integrations.Issues#8] [Ideas]
Hey all,
I think if Umbraco supported IMIS it would be used by a few people.
https://developer.imis.com/docs/accessing-the-rest-api#c-examples
At this time though I not been able to have the right settings to get this to work and I creating my own API integration to work.
It is an older CRM but used by big corporations and government services. It says it uses standards but its not quite. If anyone can have a look and provide what you feel the correct app settings would be I can re-test and then provide information for the service IF I get it working.
Thanks.

**Comment by AndyButland** [comment]
Don't mind giving it a try in the upcoming weeks, but will be tricky without an account.  Would you be willing to set up and privately share an account, a read-only request path and what you've tried so far by way of configuration?  Then perhaps we could debug through and see where it's failing or where the config needs to change,  If so I'm at _abl AT umbraco DOT dk_.
